uplink/src/base/bridge/actions_lane.rs

```diff
@@ -82,8 +81,11 @@ impl ActionsBridge {
         }
         action_status.buf_size = 1;
         streams_config.insert("action_status".to_owned(), action_status);
-        let mut streams = Streams::new(config.clone(), package_tx, metrics_tx);
+        let topic_template =
+            "/tenants/{tenant_id}/devices/{device_id}/action_status/{stream_name}".to_string();
+        let mut streams = Streams::new(config.clone(), package_tx, metrics_tx, topic_template);
         streams.config_streams(streams_config);
+        streams.max_buf_size = 1;
```

Comment:
This change is to provide similar behavior from both `action_status` and the other response streams as otherwise we will see them pushing updates once every 100 points(`MAX_BUF_SIZE`)  or 60 seconds(`DEFAULT_TIMEOUT`)